src/compas/geometry/surfaces/nurbs.py

```diff
@@ -350,6 +355,10 @@ def from_fill(cls, curve1, curve2, curve3=None, curve4=None, style="stretch"):
         """
         return new_nurbssurface_from_fill(cls, curve1, curve2, curve3, curve4, style)
 
+    @classmethod
+    def from_plane(cls, plane, u_degree=1, v_degree=1):
```

Comment:
perhaps it would make sense to add the domain. when creating a brep from a nurbssurface, domains are needed and that would make this a bit simpler...